src/Card.js

```diff
@@ -36,7 +37,7 @@ Card.propTypes = {
   boxShadowSize: PropTypes.oneOf(['sm', 'md', 'lg', 'xl']),
   borderColor: PropTypes.string,
   ...propTypes.borderRadius,
-  borderWidth: PropTypes.oneOf([1, 2])
+  borderWidth: PropTypes.oneOf([0, 1, 2])
```

Comment:
Is there actually a use case where a `Card` would have "_clear/transparent_" border?

(Confirming if the original code actually intended to force all `Card`s to have at least a 1px border.)